apps/app/src/ui/Radio.tsx

```diff
@@ -26,8 +26,8 @@ export const RadioField = forwardRef<HTMLInputElement, RadioFieldProps>(
     }
     return (
       <label className="flex items-baseline gap-4 text-left">
-        <Radio ref={ref} value={value} {...props} />
-        <div className="border-l border-border px-2 hover:border-on">
+        <Radio ref={ref} value={value} className="peer" {...props} />
```

Comment:
I don't know "peer" effect but you seems to know

Reply:
https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-sibling-state